test-harness/specs/security/security-scheme-validation-AC-8.pc.txt

```diff
@@ -0,0 +1,55 @@
+====test====
+Resource is returned when *X-API-Key* required header is provided.
+====spec====
+{
+  "info": {
+    "_postman_id": "37c4cd69-28e5-48aa-8386-a4ec010216e4"
+  },
+  "auth": {
+    "type": "apikey",
+    "apikey": [
+      {
+        "key": "in",
+        "value": "header",
+        "type": "string"
+      },
+      {
+        "key": "key",
+        "value": "X-API-Key",
+        "type": "string"
+      }
+    ],
```

Comment:
Pretty sure this is a duplicate of another test, we could move this to be a local auth — that would be valid

Reply:
Yeah we should move this to be a local authentication thing. Can we move this section down in the request?

Reply:
Which test?

This test verifies that the auth works when it is set globally (which means on a root level of PC document).

There are other tests verifying that the auth works when it is set locally or set globally and overloaded locally (including disabling).

Reply:
You are right, I must have been confused by the huge amount of tests here and there. This is ok.